test/integration/component/test_escalations_templates.py

```diff
@@ -55,23 +63,68 @@ def setUpClass(cls):
             cls.hypervisor = cls.testClient.getHypervisorInfo()
             cls.services['mode'] = cls.zone.networktype
 
+            cls.mgtSvrDetails = cls.config.__dict__["mgtSvr"][0].__dict__
+
             builtin_info = get_builtin_template_info(cls.api_client, cls.zone.id)
             cls.services["privatetemplate"]["url"] = builtin_info[0]
             cls.services["privatetemplate"]["hypervisor"] = builtin_info[1]
             cls.services["privatetemplate"]["format"] = builtin_info[2]
+            cls.services["templates"]["url"] = builtin_info[0]
+            cls.services["templates"]["hypervisor"] = builtin_info[1]
+            cls.services["templates"]["format"] = builtin_info[2]
+            if cls.zone.localstorageenabled:
+                cls.storagetype = 'local'
+                cls.services["service_offerings"][
+                    "tiny"]["storagetype"] = 'local'
+                cls.services["disk_offering"]["storagetype"] = 'local'
+            else:
+                cls.storagetype = 'shared'
+                cls.services["service_offerings"][
+                    "tiny"]["storagetype"] = 'shared'
+                cls.services["disk_offering"]["storagetype"] = 'shared'
+            cls.services["virtual_machine"]["hypervisor"] = cls.hypervisor
+            cls.services["virtual_machine"]["zoneid"] = cls.zone.id
+            cls.services["virtual_machine"]["template"] = cls.template.id
+            cls.services["custom_volume"]["zoneid"] = cls.zone.id
+            cls.service_offering = ServiceOffering.create(
+                cls.api_client,
+                cls.services["service_offerings"]["tiny"]
+            )
+            cls._cleanup.append(cls.service_offering)
         except Exception as e:
             cls.tearDownClass()
             raise Exception("Warning: Exception in setup : %s" % e)
         return
 
+    @classmethod
+    def RestartServers(cls):
+        """ Restart management server and usage server """
+
+        sshClient = SshClient(
+            cls.mgtSvrDetails["mgtSvrIp"],
+            22,
+            cls.mgtSvrDetails["user"],
+            cls.mgtSvrDetails["passwd"]
+        )
+        command = "service cloudstack-management restart"
+        sshClient.execute(command)
+        return
+    @classmethod
+    def updateConfigurAndRestart(cls,name, value):
```

Comment:
Restarting management server through test case can impact other test cases running in parallel.
Move the test to /component/maint/ folder so that it is run serially.